### PR TITLE
Introduce some goodies for making the release checking process easier

### PR DESCRIPTION
**Pull request by Altai-man**
See commit descriptions for details.

### CODE REVIEW

File: bin/blin.p6 [hunk 1]
```diff
@@ -58,17 +58,40 @@ my $skip-tests = (
    ‘Time::Duration’,
    ‘Toaster’,
    ‘Uzu’,
+   # Do not check modules that are broken and seemingly abandoned.
+   # we tried to reach out to authors, but no response for months leaves
+   # these in a limbo we don't want to spend time on
+   'Acme::Flutterby', 'Acme::Mangle', 'Acme::Skynet',
```
Comment (AlexDaniel) [inline bin/blin.p6]: I'm not sure about this. The output produced by blin is used from time to time to populate https://github.com/perl6/ecosystem-unbitrot (which creates/closes tickets as needed). What's the disadvantage in testing modules that are known to always fail, unless they're flappers?

File: bin/blin.p6 [hunk 2]
```diff
@@ -58,17 +58,40 @@ my $skip-tests = (
...
+   'AI::FANN', 'Algorithm::BloomFilter', 'Algorithm::Viterbi',
+   'API::USNavalObservatory', 'App::jsonv', 'App::P6Dx',
+   'Aria', 'Astro::Sunrise', 'Backtrace::AsHTML', 'Bencode',
+   'Class::Utils', 'CompUnit::Search', 'CompUnit::Util',
+   'Data::Pretty', 'Editsrc::Uggedit', 'Facter', '	FileSystem::Capacity',
```
Comment (AlexDaniel) [inline bin/blin.p6]: There's a TAB character here.

File: bin/blin.p6 [hunk 3]
```diff
@@ -259,7 +282,10 @@ note ‘🥞🥞 Marking latest versions and their deps’;
 for %lookup {
     next unless .key eq .value».name.any; # proceed only if not an alias
     if @specified-modules or $custom-script {
-        next if not .key eq @specified-modules.any | $custom-script.any;
+        next if not .key eq @specified-modules.any;
+        with $custom-script {
+            next if not .key eq $_.any;
```
Comment (AlexDaniel) [inline bin/blin.p6]: This can't be right? `with` will change `$_` and `.key` is going to do the wrong thing.

File: bin/blin.p6 (hunk 2 above)
Comment (Altai-man) [reply bin/blin.p6]: Thanks! That's a catch...

File: bin/blin.p6 (hunk 1 above)
Comment (Altai-man) [reply bin/blin.p6]: These modules were abandoned by authors for different reasons.
We did send issues, we did write patches and PR them. Then months and months passed and no action was taken.
They add clutter to release process. You have to check failures when cutting a release, and you have a big bunch of modules which are just broken. You don't remember each one of them and for each one you are checking if the author is aware, if there is an issue, "Oh, I saw that already a half of year ago, nothing has changed, just a false positive I spent time on".
When there are 10 broken modules that's ok, but if you have more, it is not so easy anymore.
I don't see any merit in creating new tickets from this list to ecosystem-unbitrot repo. It was there already. It didn't work out after all. The disadvantage is time spent. The advantage is... I don't see one, really.

File: bin/blin.p6 (hunk 3 above)
Comment (Altai-man) [reply bin/blin.p6]: D'oh.

**Comment by Altai-man** [comment]
I don't want to fight for the controversial commit, so a revert.
Other commits should be useful enough to merge.

**Comment by AlexDaniel** [comment]
Thanks!

**Comment by Altai-man** [comment]
@AlexDaniel do we need to do something for docker via dblin to match up those changes?

**Pull request merged**